api/orchestration_movers.go

```diff
@@ -452,3 +452,32 @@ func (o *datasyncOrchestrator) taskDetailsFromName(ctx context.Context, group, n
 
 	return nil, nil, apierror.New(apierror.ErrNotFound, "datasync mover not found", nil)
 }
+
+// taskRunsFromTaskArn finds a datasync task runs  based on its group/name and returns information about it
+func (o *datasyncOrchestrator) taskRunsFromTaskArn(ctx context.Context, taskArn string) ([]string, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	// get a list of all datasync resources in the group
+	out, err := o.datasyncClient.ListDatasyncTaskExecutions(ctx, taskArn)
+	if err != nil {
+		return nil, err
+	}
+
+	return out, nil
+}
+
+func (o *datasyncOrchestrator) TaskDetailsFromid(ctx context.Context, id string) (*datasync.DescribeTaskExecutionOutput, error) {
+	if id == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	out, err := o.datasyncClient.DescribeTaskExecution(ctx, id)
```

Comment:
This id will be just `exec-123456789`, so we need to construct the full exec ARN before calling this

Reply:
Please review.